magenta/models/nsynth/wavenet/generate.py

```diff
@@ -0,0 +1,146 @@
+import tensorflow as tf
+from scipy.io import wavfile
+from magenta.models.nsynth import utils
+from magenta.models.nsynth.wavenet.h512_bo16 import Config, FastGenerationConfig
+import numpy as np
+
+
+def inv_mu_law(x, mu=255.0):
+  """A numpy implementation of inverse Mu-Law.
+  Args:
+    x: The Mu-Law samples to decode.
+    mu: The Mu we used to encode these samples.
+  Returns:
+    out: The decoded data.
+  """
+  x = np.array(x).astype(np.float32)
+  out = (x + 0.5) * 2. / (mu + 1)
+  out = np.sign(out) / mu * ((1 + mu)**np.abs(out) - 1)
+  out = np.where(np.equal(x, 0), x, out)
+  return out
+
+
+def load_audio(wav_file, sample_length=64000):
```

Comment:
These functions should probably be loaded from / added to utils.py. You could just rename them as inv_mu_law_numpy() for example.